### PR TITLE
janitor wall closet.

### PR DESCRIPTION
**Pull request by moonheart08**
## About the PR
it's like a janitor closet but for walls

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Storage/Closets/wall_lockers.yml [hunk 1]
```diff
@@ -153,6 +153,12 @@
       state_open: generic_open
       state_closed: mixed_door
 
+- type: entity
+  id: ClosetWallJanitor
+  parent: ClosetWallMixed
+  name: custodial closet
+  description: It's a storage unit for janitorial clothes and gear.
```
Comment (theashtronaut) [inline Resources/Prototypes/Entities/Structures/Storage/Closets/wall_lockers.yml:160]:
```suggestion
  description: It's a storage unit for janitorial supplies.
```
it doesn't have any clothes in it :p

**Comment by mirrorcult** [comment]
i feel like the fill should just inherit from the custodial closet so we dont get cases where people forget to update both

**Comment by EmoGarbage404** [comment]
> i feel like the fill should just inherit from the custodial closet so we dont get cases where people forget to update both

The closets cant inherit from each other because they will then inherit anchorable and other stuff that will break it. Perhaps either make storage fills prototypes or make abstract entities?

**Comment by ShadowCommander** [comment]
The StorageFill can be moved to a separate entity and use prototype composition on both the locker and wall closet.

**Review by ShadowCommander — CHANGES_REQUESTED** [review]
The StorageFill can be moved to a separate entity and use prototype composition on both the locker and wall closet.

**Comment by mirrorcult** [comment]
oh yeah that exists now

**Pull request closed without merging**